src/Block/Breadcrumb/BaseBreadcrumbMenuBlockService.php

```diff
@@ -86,4 +121,32 @@ protected function getRootMenu(BlockContextInterface $blockContext): ItemInterfa
 
         return $menu;
     }
+
+    /**
+     * Replaces setting keys with knp menu item options keys.
+     *
+     * @param array<string, mixed> $settings
+     *
+     * @return array<string, mixed>
+     */
+    private function getMenuOptions(array $settings): array
+    {
+        $mapping = [
+            'current_class' => 'currentClass',
+            'first_class' => 'firstClass',
+            'last_class' => 'lastClass',
+            'safe_labels' => 'allow_safe_labels',
+            'menu_template' => 'template',
+        ];
+
+        $options = [];
+
+        foreach ($settings as $key => $value) {
+            if (\array_key_exists($key, $mapping) && null !== $value) {
```

Comment:
Minor performance improvement.
```suggestion
            if (null !== $value && \array_key_exists($key, $mapping)) {
```